src/app/core/entity-components/entity-details/form/form.stories.ts

```diff
@@ -0,0 +1,97 @@
+import { FormComponent } from "./form.component";
+import { moduleMetadata } from "@storybook/angular";
+import { Meta, Story } from "@storybook/angular/types-6-0";
+import { EntityMapperService } from "../../../entity/entity-mapper.service";
+import { AlertService } from "../../../alerts/alert.service";
+import { ChildPhotoService } from "../../../../child-dev-project/children/child-photo-service/child-photo.service";
+import { SessionService } from "../../../session/session-service/session.service";
+import { EntityDetailsModule } from "../entity-details.module";
+import { Child } from "../../../../child-dev-project/children/model/child";
+import { RouterTestingModule } from "@angular/router/testing";
+import { User } from "../../../user/user";
+import { BrowserAnimationsModule } from "@angular/platform-browser/animations";
+import { EntityPermissionsService } from "../../../permissions/entity-permissions.service";
+
+export default {
+  title: "Core/Form",
+  component: FormComponent,
+  decorators: [
+    moduleMetadata({
+      imports: [
+        EntityDetailsModule,
+        RouterTestingModule,
+        BrowserAnimationsModule,
+      ],
+      providers: [
+        { provide: EntityMapperService, useValue: {} },
+        { provide: AlertService, useValue: {} },
+        { provide: ChildPhotoService, useValue: { canSetImage: () => true } },
+        {
+          provide: SessionService,
+          useValue: { getCurrentUser: () => new User() },
+        },
+        {
+          provide: EntityPermissionsService,
+          useValue: { userIsPermitted: () => false },
```

Comment:
can we built in a tooltip to tell users that the button is deactivated because they don't have required permissions?

Reply:
Done